saleor/dashboard/product/forms.py

```diff
@@ -56,6 +57,11 @@ def __init__(self, *args, **kwargs):
         field.widget.attrs['data-placeholder'] = pgettext_lazy(
             'Product form labels', 'Search')
 
+    def save(self, *args, **kwargs):
+        instance = super(ProductForm, self).save(*args, **kwargs)
+        update_product(instance)
```

Comment:
I would give a more descriptive name to this method, e.g. `update_product_index`, as it's a bit confusing in the context of form's `save` method which does the actual update of the product instance.